electrum/simple_config.py

```diff
@@ -1047,6 +1047,7 @@ def __setattr__(self, name, value):
     )
 
     LIGHTNING_NODE_ALIAS = ConfigVar('lightning_node_alias', default='', type_=str)
+    LIGHTNING_FORWARD_ONION_MESSAGES = ConfigVar('lightning_forward_onion_messages', default=False, type_=bool)
```

Comment:
Not sure if we've ever linked config vars together, but perhaps it is time :D
```suggestion
    LIGHTNING_FORWARD_ONION_MESSAGES = ConfigVar('lightning_forward_onion_messages', default=lambda config: config.EXPERIMENTAL_LN_FORWARD_PAYMENTS, type_=bool)
```
As in, if someone has enabled payment forwarding but have not touched this config var, we could default enable this too.

Reply:
For a moment there I thought I committed a copypaste error :D

Reply:
> As in, if someone has enabled payment forwarding but have not touched this config var, we could default enable this too.

This will probably lead to confusion somewhere.. We could use a single config option for forwarding in general, payments and messages?

Reply:
> confusion

maybe..

Well in that case I would prefer to just keep the current "EXPERIMENTAL_LN_FORWARD_PAYMENTS" config var as-is, and reuse it for onion msg forwarding. No need to rename it I mean, I think.

Reply:
renaming it is a PITA, as we then need to port older config keys.. :(

Reply:
Please do not create unneeded config variables. I do not think there is any scenario where we would want to forward payments and not onion messages. And if we need to do that one day, then it will be time to define a new config variable. But for now, please use the current config var.